helix-view/src/editor.rs

```diff
@@ -155,7 +155,7 @@ impl Editor {
             let language_server = doc
                 .language
                 .as_ref()
-                .and_then(|language| self.language_servers.get(language));
+                .and_then(|language| self.language_servers.get(language).ok());
```

Comment:
This is better than the old behavior (crashing), but we need to find a way to log LSP errors to the UI (`set_error` isn't available here)